main_test.go

```diff
@@ -561,12 +566,13 @@ func TestEventRPCNotifier(t *testing.T) {
 func TestSubscribeCollationWithRPCNotifier(t *testing.T) {
```

Comment:
nitpick: `RPCNotifier` or `EventNotifier`?

Reply:
Changed it to `RPCEventNotifier`